app/helpers/data_getter.py

```diff
@@ -216,16 +216,15 @@ def get_tracks_by_event_id():
         return Track.query.filter_by(event_id=get_event_id())
 
     @staticmethod
-    def get_sessions(event_id, state='accepted'):
```

Comment:
@niranjan94 
what do you suggest ?
If I keep that parameter then how do I pass two parameters to check with , cause in scheduler it should have both types of states **confirmed** as well as **accepted**.
And also we can't pass state as an array cause it may again break somewhere.